gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1155,12 +1155,18 @@ gst_tensor_converter_parse_caps (GstTensorConverter * self,
       break;
     }
     case _NNS_TEXT:
+      /* get fixed size of text string from property */
+      if (!gst_tensor_dimension_is_valid (self->tensor_info.dimension)) {
+        GST_ERROR_OBJECT (self,
+            "Failed to get tensor info, update string size.\n");
```

Comment:
Please tell the user that he/she needs to add a property value of "input-dim" to set max string size.